packages/parany/parany.4.0.0/opam

```diff
@@ -0,0 +1,20 @@
+opam-version: "1.2"
+maintainer: "Francois BERENGER <ligand@free.fr>"
+authors: ["Francois Berenger"]
+homepage: "https://github.com/UnixJunkie/parany"
+bug-reports: "https://github.com/UnixJunkie/parany/issues"
+dev-repo: "https://github.com/UnixJunkie/parany.git"
+license: "LGPL"
+build: [
+  ["ocaml" "setup.ml" "-configure" "--prefix" prefix]
+  ["ocaml" "setup.ml" "-build"]
+]
+install: ["ocaml" "setup.ml" "-install"]
+remove: ["ocamlfind" "remove" "parany"]
+depends: [
+  "ocamlfind"
```

Comment:
Could you add the ```{build}``` attribute to this dependency unless you are depending on the ```Findlib``` library ?